test/GroupByTest.cpp

```diff
@@ -1527,9 +1531,7 @@ TEST_F(GroupByOptimizations, computeGroupByForJoinWithFullScan) {
         chooseInterface
             ? validForOptimization.computeGroupByForJoinWithFullScan()
             : validForOptimization.computeOptimizedGroupByIfPossible();
-    EXPECT_THAT(
-        optional,
-        Optional(Eq(W{makeIdTableFromVector({{idOfX, I(7)}, {idOfY, I(1)}})})));
+    EXPECT_THAT(optional, optionalTable({{idOfX, I(7)}, {idOfY, I(1)}}));
```

Comment:
I believe the name `optionalTable` implies that the table is optional, not that it's a table wrapped inside an optional. So perhaps this should be renamed to something like `nonOptionalTable`? Or `tableInOptional`? Or maybe something that has more flow with the entire "assert that" naming `optionalHasTable`?